public/js/angular/filters.js

```diff
@@ -30,14 +30,11 @@ angular
       if (likes) {
         var likesText;
 
-        switch (likes.length) {
-        case 0:
+        if (likes.length === 0) {
```

Comment:
I don't understand, this is perfectly readable. If you know JS, you know this says "we need like-n, except for 0 and 1".